config/routes.rb

```diff
@@ -1,5 +1,6 @@
 Rails.application.routes.draw do
- 	get "/help", to: "static_pages#help"
+  get "users/new"
```

Comment:
không chị ạ, do khi generate controller sinh ra tự động view new để thêm mới người dùng, nhưng do chap trước đã có route 'get "/signup", to: "users#new"' nên không cần route này nữa. deleted.